wsclient/src/test/java/org/hpccsystems/ws/client/WSSQLClientTest.java

```diff
@@ -91,14 +95,17 @@ public void ping() throws Exception
     @Test
     public void testGetTables() throws Exception
     {
+        if (mockTableName != null && !mockTableName.isEmpty())
```

Comment:
Why do we need the check here? Was another test setting mockTableName? Does that invalidate the current test if that is the case?